go.mod

```diff
@@ -22,12 +21,10 @@ require (
 	github.com/fsouza/go-dockerclient v1.3.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/gogo/protobuf v1.2.0 // indirect
+	github.com/golangci/golangci-lint v1.17.1
```

Comment:
This should be moved over ot https://github.com/containers/buildah/tree/master/tests/tools .

Reply:
Done :+1: